assets/ui/latinQuotes.ui

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
We follow a 4 space indentation. Please find-replace all 2 spaces with 4 spaces. You should also configure your IntelliJ IDEA to use the Terasology style settings.